functions/go/starlark/config.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	"sigs.k8s.io/kustomize/kyaml/fn/runtime/runtimeutil"
+	"sigs.k8s.io/kustomize/kyaml/fn/runtime/starlark"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+const (
+	fnConfigGroup      = "kpt.dev"
```

Comment:
Let's follow offline to standardize the group and version for all function configs. Using the kpt CLI group doesn't seem right.

@phanimarupaka

Reply:
Maybe we can use `fn.kpt.dev`?